tests/pytype_test.py

```diff
@@ -119,16 +164,21 @@ def pytype_test(args):
     while 1:
         while files and len(running_tests) < args.num_parallel:
             f = files.pop()
+            run_cmd = [
+                pytype_exe,
+                '--typeshed-location=%s' % dirs.typeshed,
+                '--module-name=%s' % _get_module_name(f),
+                '--convert-to-pickle=%s' % os.devnull
+            ]
```

Comment:
Move this below to where it's used?

Reply:
it's used just below; the if block appends to it.

Reply:
see below